stix_shifter/stix_transmission/src/modules/stix_bundle/stix_bundle_connector.py

```diff
@@ -42,7 +42,16 @@ def match(self, pattern, observed_data_sdos, verbose=False):
         return matching_sdos
 
     def ping(self):
-        return {"success": True}
+        return_obj = dict()
+
+        response = self.call_api('head')
+        response_code = response.status_code
+
+        if response_code == 200 or response_code == 301:
+            return_obj['success'] = True
+        else:
+            ErrorResponder.fill_error(return_obj, ['message'])
```

Comment:
you seem to be missing `response` as the second parameter for `ErrorResponder.fill_error`

Reply:
right. updated